src/NCronJob/Configuration/Runtime/RuntimeServiceCollection.cs

```diff
@@ -9,8 +9,8 @@ namespace NCronJob;
 internal sealed class RuntimeServiceCollection : List<ServiceDescriptor>, IServiceCollection
 {
     public ImmutableArray<JobDefinition> GetJobDefinitions()
-        => [..this.Select(s => s.ImplementationInstance).OfType<JobDefinition>()];
+        => [..this.Select(s => s.ImplementationInstance).OfType<JobRegistry>().Single().JobDefinitions];
 
     public ImmutableArray<DynamicJobRegistration> GetDynamicJobFactoryRegistries()
```

Comment:
This is actually never used. Is that a leftover or should this be leveraged somewhere (tests, ...)?

Reply:
It was used inside the RuntimeJobRegistry. As you can add "dynamic" jobs (`AddJob(() => {}, ....`)

Reply:
`GetJobDefinitions` have been dropped